trends_presentation/src/commonMain/kotlin/net/thechance/mena/trends/presentation/screen/manage_my_trends/ManageTrendsScreen.kt

```diff
@@ -104,8 +108,10 @@ private fun ManageTrendsScreenContent(
             )
         },
         content = {
+            val reels = state.reels.collectAsLazyPagingItems()
+
             TrendsAnimatedVisibility(
-                visible = state.isLoading,
+                visible = reels.loadState.refresh is LoadState.Loading,
```

Comment:
It might be a good idea to add an extension function for `loadState.isLoading()` to make it more readable.